doc/source/development/contributing.rst

```diff
@@ -157,9 +157,56 @@ Instead of manually setting up a development environment, you can use Docker to
+When the whale icon in the status bar stays steady, Docker Desktop is up-and-running, and is accessible from any terminal window.
+
+See  `official docker documentation <https://docs.docker.com/docker-for-windows/install-windows-home/>`_ for details.
+
+**Installing Docker on Windows Pro, Enterprise, or Education**
+
+First make sure that you have Windows 10 64-bit: Pro, Enterprise, or Education (Build 15063 or later). Ensure that Hyper-V feature is `enabled <https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v>`_.
+
+Download and install `Docker for Windows <https://docs.microsoft.com/pl-pl/windows/wsl/wsl2-kernel>`_.
+When prompted, ensure the `Enable Hyper-V Windows Features` option is selected on the Configuration page.
+If your admin account is different to your user account, you must add the user to the docker-users group. Run Computer Management as an administrator and navigate to `Local Users and Groups > Groups > docker-users`. Right-click to add the user to the group. Log out and log back in for the changes to take effect.
+Docker Desktop does not start automatically after installation. To start Docker Desktop, search for Docker, and select Docker Desktop in the search results.
+When the whale icon in the status bar stays steady, Docker Desktop is up-and-running, and is accessible from any terminal window.
+
+See `official docker documentation <https://docs.docker.com/docker-for-windows/install/>`_ for details.
+
+**Installing Docker on Unix**
+
+ See official docker documentation for detailed information on `installing <https://docs.docker.com/engine/install/>`_ Docker in various Linux distributions  and `post-installation steps <https://docs.docker.com/engine/install/linux-postinstall/>`_.
+
+**Installing Docker on Mac OS**
+
+See `official docker documentation <https://docs.docker.com/docker-for-mac/install/>`_ for details.
+
+**Test your Docker installation**
+
+Run the following command in your terminal::
+
+    docker run hello-world
+
+You should see a welcome message, your installation is complete! In case of problems refer to these pages with common issues on: `Windows <>https://docs.docker.com/docker-for-windows/troubleshoot/`_, 
+`MacOS <https://docs.docker.com/docker-for-mac/troubleshoot/>`_.
+
+**Development inside Docker container**
+
+You can launch a remote session with Visual Studio Code, a popular free IDE, using the `.devcontainer.json` file in root directory. To do that you need to 
+Install `Visual Studio Code <https://code.visualstudio.com/insiders/>`_ and the `Remote Development extension pack <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack>`_. Then follow these `steps <https://code.visualstudio.com/docs/remote/containers#_quick-start-try-a-dev-container>`_ to connect to remote session.
```

Comment:
Should the link be "Try a dev container" or https://code.visualstudio.com/docs/remote/containers#_quick-start-open-an-existing-folder-in-a-container ?

Reply:
You're right, makes more sense. It's fixed now.